mmpose/core/post_processing/temporal_filters/one_euro_filter.py

```diff
@@ -0,0 +1,110 @@
+# ------------------------------------------------------------------------------
+# Adapted from https://github.com/HoBeom/OneEuroFilter-Numpy
+# Original licence: Copyright (c)  HoBeom Jeon, under the MIT License.
+# ------------------------------------------------------------------------------
+import math
+
+import numpy as np
+
+from .builder import FILTERS
+from .filter import TemporalFilter
+
+
+def smoothing_factor(t_e, cutoff):
+    r = 2 * math.pi * cutoff * t_e
+    return r / (r + 1)
+
+
+def exponential_smoothing(a, x, x_prev):
+    return a * x + (1 - a) * x_prev
+
+
+class OneEuro:
+
```

Comment:
Do we need any description here?

Reply:
This is part of the newly reimplemented One-Euro filter. The one that will be deprecated is in `mmpose/core/post_processing/one_euro_filter.py`.